src/components/organisms/AdminVenueView/components/MapBackgroundPlaceholder/MapBackgroundPlaceholder.scss

```diff
@@ -0,0 +1,13 @@
+@import "scss/constants.scss";
+
+.MapBackgroundPlaceholder {
+  position: relative;
+  width: 100%;
+  height: 80vh;
+  border-radius: $border-radius--lg;
+  box-shadow: $venue-rooms-editor-background-placeholder-box-shadow;
+  background: $venue-rooms-editor-background;
```

Comment:
Since this is no longer venue room specific, these values can be renamed.

Reply:
These values are specific to `<MapBackgroundPlaceholder/>` and I moved them into the relevant `.scss` file

Fixed in 47307d8782d92ea2e154257b2d179cd84423861e